viral/ensemble_reactivation.py

```
@@ -1259,6 +1266,78 @@ def all_mice_ensemble_results_plots(
     plt.xlabel("Time (s) from event onset")
 
 
+def compare_run_results(W_py: np.ndarray, W_mat: np.ndarray) -> None:
```

Ruff (0.13.3)

Redefinition of unused `compare_run_results` from line 1097

(F811)